admin/broadleaf-open-admin-platform/src/main/java/org/broadleafcommerce/openadmin/dto/ParentRecordStructure.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * #%L
+ * BroadleafCommerce Open Admin Platform
+ * %%
+ * Copyright (C) 2009 - 2019 Broadleaf Commerce
+ * %%
+ * Licensed under the Broadleaf Fair Use License Agreement, Version 1.0
+ * (the "Fair Use License" located  at http://license.broadleafcommerce.org/fair_use_license-1.0.txt)
+ * unless the restrictions on use therein are violated and require payment to Broadleaf in which case
+ * the Broadleaf End User License Agreement (EULA), Version 1.1
+ * (the "Commercial License" located at http://license.broadleafcommerce.org/commercial_license-1.1.txt)
+ * shall apply.
+ * 
+ * Alternatively, the Commercial License may be replaced with a mutually agreed upon license (the "Custom License")
+ * between you and Broadleaf Commerce. You may not use this file except in compliance with the applicable license.
+ * #L%
+ */
+package org.broadleafcommerce.openadmin.dto;
+
+import org.broadleafcommerce.openadmin.dto.visitor.PersistencePerspectiveItemVisitor;
+
+import java.io.Serializable;
+import java.util.Arrays;
+
+public class ParentRecordStructure implements Serializable, PersistencePerspectiveItem {
+
+    private static final long serialVersionUID = 1L;
+
+    private Entity parentRecord;
+    private ClassMetadata mainMetadata;
```

Comment:
I think you should call this `parentMetadata`. It should match the same type of name as `parentRecord`.